spaceship/lib/spaceship/testflight/base.rb

```diff
@@ -1,7 +1,7 @@
-module Testflight
+module TestFlight
```

Comment:
we should move the `testflight` folder to `test_flight` if the module name is `TestFlight`